plugins/custom-cols.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: custom-cols
+spec:
+  version: "v0.0.2"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/webofmars/kubectl-custom-cols/archive/v0.0.3.tar.gz
+    sha256: "f83832f5ee0a3e3b44984f29e9ea7ef5b380e23d53da6d52b4a7d62d925ad9f6"
+    files:
+    - from: "./kubectl-custom-cols-*/kubectl-custom-cols"
+      to: "."
+    - from: "./kubectl-custom-cols-*/templates/*.tpl"
+      to: "templates"
+    bin: "./kubectl-custom-cols"
+  shortDescription: Easy use of custom columns output formating.
```

Comment:
As @.ahmetb suggested, you could mention `kubectl get` replacement in the short description. I think this will increase the discoverability.

Reply:
Right, i replaced it in the main repo but not here. FIXED.